src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -56,6 +56,7 @@ enum class StatusCode {
 }
 
 private val PROJECT_ID = ServiceOptions.getDefaultProjectId()
+private val MOCK_PDF_FILE = Paths.get("src","main","resources", "example.pdf").toFile()
```

Comment:
I don't think that you will have such path in production. More reliable way is reading data from the InputStream created by [getResource method in ClassLoader](https://docs.oracle.com/javase/7/docs/api/java/lang/ClassLoader.html#getResourceAsStream(java.lang.String))

Reply:
Done